app/scripts/templates/reset_password.mustache

```diff
@@ -1,31 +1,27 @@
 <div id="main-content" class="card">
   <header>
-    <h1 id="fxa-reset-password-header">{{#t}}Reset password{{/t}}</h1>
+    <h1 id="fxa-reset-password-header">{{#t}}Forgot password{{/t}}</h1>
   </header>
 
   <section>
     <div class="error"></div>
 
     <form novalidate>
-      <p>{{#t}}Are you sure you want to reset your account password?{{/t}}</p>
-
-      <p class="reset-warning">
-        {{#t}}Resetting your password will erase your browsing data from our servers. Any data on your devices will remain untouched. Begin the reset process by entering your email below.{{/t}}
-      </p>
-
+        <p class="reset-warning">
+            {{#t}}When you reset your password, you will lose any Sync data that is not on one of your devices.{{/t}}<a href="https://support.mozilla.org/en-US/products/firefox/sync">{{#t}} Learn how Sync works.{{/t}}</a>
+        </p>
       <div class="input-row">
         <input name="email" type="email" class="email" placeholder="{{#t}}Email{{/t}}" value="{{ email }}" spellcheck="false" autofocus required>
       </div>
 
       <div class="button-row">
-        <button id="submit-btn" type="submit" class="disabled">{{#t}}Begin reset{{/t}}</button>
+        <button id="submit-btn" type="submit" class="disabled">{{#t}}Reset Password{{/t}}</button>
       </div>
     </form>
 
-    {{#canGoBack}}
-      <div class="links">
-        <a id="back" href="#">{{#t}}Back{{/t}}</a>
```

Comment:
If `Back` is no longer a thing here, can you remove the "BackMixin" from the view?